processor/logdedupprocessor/README.md

```diff
@@ -15,7 +15,7 @@ This processor is used to deduplicate logs by detecting identical logs over a ra
 
 ## How It Works
 1. The user configures the log deduplication processor in the desired logs pipeline.
-2. All logs sent to the processor and aggregated over the configured `interval`. Logs are considered identical if they have the same body, resource attributes, severity, and log attributes.
+2. Each log entry for which any of the `conditions` evaluates `true` is aggregated over the configured `interval`. Logs that do not match any condition in `conditions` are passed onward in the pipeline without aggregating. Logs are considered identical if they have the same body, resource attributes, severity, and log attributes.
```

Comment:
We need to update this so it indicates if that if no conditions are provided (default), all log entries will be aggregated.

Maybe something like:

```suggestion
2. If the processor does not provide `conditions`, all logs are considered eligible for aggregation. If the processor does have configured `conditions`, all log entries where at least one of the `conditions` evaluates `true` is considered eligible for aggregation. Eligible logs are aggregated over the configured `interval`. Logs that do not match any condition in `conditions` are passed onward in the pipeline without aggregating. Logs are considered identical if they have the same body, resource attributes, severity, and log attributes.
```

Reply:
Thanks Mike, just updated the readme with something inspired by your suggestion 👌